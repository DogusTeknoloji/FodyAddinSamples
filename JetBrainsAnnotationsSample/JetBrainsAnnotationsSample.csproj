<?xml version="1.0" encoding="utf-8"?>
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFrameworks>net471</TargetFrameworks>
    <Tags>TE</Tags>
  </PropertyGroup>

  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net47|AnyCPU'">
    <DefineConstants>TRACE;DEBUG;JETBRAINS_ANNOTATIONS</DefineConstants>
  </PropertyGroup>

  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net47|AnyCPU'">
    <DefineConstants>TRACE;JETBRAINS_ANNOTATIONS</DefineConstants>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Fody" Version="3.0.3" />
    <PackageReference Include="JetBrainsAnnotations.Fody" Version="2.2.0" />
    <PackageReference Include="JetBrains.Annotations" Version="11.1.0" />
    <PackageReference Include="xunit" Version="2.3.1" />
    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.2" />
  </ItemGroup>

</Project>
