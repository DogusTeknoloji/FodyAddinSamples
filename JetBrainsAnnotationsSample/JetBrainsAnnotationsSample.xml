<?xml version="1.0" encoding="utf-8"?>
<doc>
    <assembly>
        <name>JetBrainsAnnotationsSample</name>
    </assembly>
    <members>
        <member name="T:JetBrainsAnnotationsSample">
            <summary>
            The sample class.
            </summary>
        </member>
        <member name="M:JetBrainsAnnotationsSample.Run">
            <summary>
            Runs the tests.
            </summary>
        </member>
        <member name="M:JetBrainsAnnotationsSample.AnyMethod(System.Object)">
            <summary>
            This is an annotated method.
            <p /><c>[CanBeNull]</c></summary>
            <param name="anyParam">Any parameter.<p /><c>[NotNull, UsedImplicitly]</c></param>
            <returns>something</returns>
        </member>
        <member name="P:JetBrainsAnnotationsSample.AnyProperty">
            <summary>
            Gets or sets the annotated property.
            <p /><c>[NotNull]</c></summary>
        </member>
        <member name="M:JetBrainsAnnotationsSample.Format(System.String,System.Object[])">
            <summary>
            String format annotated.
            <p /><c>[NotNull]</c></summary>
            <param name="format">The format.<p /><c>[NotNull]</c></param>
            <param name="args">The arguments.<p /><c>[NotNull, ItemNotNull]</c></param>
            <returns>The result.</returns>
        </member>
    </members>
</doc>
