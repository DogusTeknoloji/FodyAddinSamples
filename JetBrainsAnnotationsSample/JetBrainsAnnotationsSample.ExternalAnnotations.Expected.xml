<?xml version="1.0" encoding="utf-8"?>
<assembly name="JetBrainsAnnotationsSample">
  <member name="M:JetBrainsAnnotationsSample.AnyMethod(System.Object)">
    <attribute ctor="M:JetBrains.Annotations.CanBeNullAttribute.#ctor" />
    <parameter name="anyParam">
      <attribute ctor="M:JetBrains.Annotations.NotNullAttribute.#ctor" />
      <attribute ctor="M:JetBrains.Annotations.UsedImplicitlyAttribute.#ctor" />
    </parameter>
  </member>
  <member name="P:JetBrainsAnnotationsSample.AnyProperty">
    <attribute ctor="M:JetBrains.Annotations.NotNullAttribute.#ctor" />
  </member>
  <member name="M:JetBrainsAnnotationsSample.Format(System.String,System.Object[])">
    <attribute ctor="M:JetBrains.Annotations.NotNullAttribute.#ctor" />
    <attribute ctor="M:JetBrains.Annotations.StringFormatMethodAttribute.#ctor(System.String)">
      <argument>format</argument>
    </attribute>
    <parameter name="format">
      <attribute ctor="M:JetBrains.Annotations.NotNullAttribute.#ctor" />
    </parameter>
    <parameter name="args">
      <attribute ctor="M:JetBrains.Annotations.NotNullAttribute.#ctor" />
      <attribute ctor="M:JetBrains.Annotations.ItemNotNullAttribute.#ctor" />
    </parameter>
  </member>
</assembly>